skfda/exploratory/outliers/_directional_outlyingness.py

```diff
@@ -2,15 +2,16 @@
 
 from numpy import linalg as la
 import scipy.integrate
-from scipy.stats import f, variation
+from scipy.stats import f
+import scipy.stats
 from sklearn.base import BaseEstimator, OutlierMixin
 from sklearn.covariance import MinCovDet
 
 import numpy as np
-from skfda.exploratory.depth import modified_band_depth
+from skfda.exploratory.depth.multivariate import projection_depth
 
+from . import _directional_outlyingness_experiment_results as experiments
```

Comment:
The values of the experiments are only used in ``_parameters_numeric ``, maybe this import could be moved to the function, in order to avoid load the module with the data.

Reply:
Fixed in 25a09c57a3a259d4a4df9506b6f4af06e4d24ad5.